tooling/greybox_fuzzer/src/mutation/configurations.rs

```diff
@@ -0,0 +1,874 @@
+use rand::Rng;
```

Comment:
The logic for selecting a mutation based on weights inside a configuration is repeated here a lot. I was thinking how this could be avoided and came up with this: https://gist.github.com/asterite/cbea91f23ec600be288bcc39b0a3d14e

The only downside to it that I see is that you might forget to include an option in the config... however, I think that downside also exists in the current code: if you add a new option you must remember to add it to the configuration, and you must also remember to add to the total weight.

Reply:
Changed all of that